pkg/agent/flowexporter/types.go

```diff
@@ -76,3 +79,23 @@ type FlowRecord struct {
 	LastExportTime     time.Time
 	IsActive           bool
 }
+
+type DenyConnection struct {
+	FlowKey                        Tuple
+	PacketLength                   uint64
```

Comment:
We should use `Bytes` or `FlowBytes` here to keep it consistent with the existing connection type.